packages/components/src/modal/aria-helper.js

```diff
@@ -32,8 +27,8 @@ export function hideApp( unhiddenElement ) {
 	if ( isHidden ) {
 		return;
 	}
-	const elements = document.body.children;
-	forEach( elements, ( element ) => {
+	const elements = Array.from( document.body.children );
```

Comment:
TIL `document.body.children` doesn't have a `forEach` method...

Reply:
Yeah, `HTMLCollection` is limited to having `.length`, but doesn't reliably implement the iterator protocol in all browsers. Some browser support some means of iteration (like `for/of` and `for/in` for example) but not reliably, so `Array.from()` ensures we'll be able to iterate the collection of elements.